laboratory-servlet-api-integration/src/it/java/com/wixpress/petri/test/LaboratoryIT.java

```diff
@@ -141,4 +140,13 @@ public void reportsSingleSpecToPetri() throws Exception {
         assertThat(specs.get(0).getScopes(), CoreMatchers.is(Arrays.asList(ValidStubSpecDefinition_1.scopeDefinitions)));
     }
 
+    @Test
+    public void resilientToPetriServerDowntime() throws Exception {
```

Comment:
this doesnt really test anything. the setupExperiment cause a fetch of experiments. next request failing means nothing (will happen only in 5 minutes).

attaching the 2 test we originally had that duplicated some koboshi logic.
must say im not sure which is appropriate here:

```
// This test validates the contract with the rdf is kept
    // i.e previous data is kept if exception is thrown while fetching
    // (in theory should not be our concern, but being OCD about this..)
    @Test
    public void retainsStaleVersionWhenRdfFails() {
        remoteDataSourceDriver.internalInsertExperiment(experimentWithWinningFirstGroupWithEditorScope().build());
        assertThat(experimentsStorage.findById(1), is(notNullValue()));

        ramPetriClient.setBlowUp(true);
        remoteDataSourceDriver.internalInsertExperiment(experimentWithWinningFirstGroupWithEditorScope().build());

        assertThat(experimentsStorage.findById(2), is(nullValue()));
        assertThat(experimentsStorage.findById(1), is(notNullValue()));
    }

    // TODO - probably remove this test once rdf api change is complete...
    // This test validates the contract with the rdf is kept
    // i.e previous data is kept if exception is thrown while fetching + staleness is marked
    // (in theory should not be our concern, but being OCD about this..)
    @Test
    public void doesNotCleanCookiesWhenRdfFails() {
        remoteDataSourceDriver.internalInsertExperiment(experimentWithWinningFirstGroupWithEditorScope().build());
        ramPetriClient.setBlowUp(true);
        remoteDataSourceDriver.fetchNow();
        WixApiResponse wixApiResponse = conductSingleExperimentWithAnonCookie("2#1");
        assertThat(anonymousABTestCookieValue(wixApiResponse), is("2#1|1#1"));
    }
```